CONTRIBUTING.md

```diff
@@ -40,6 +40,8 @@ This will create a new folder `browser/node<node version>-chrome<chrome version>
 
 **note:** The Edge browser will always default to the latest stable release. There is currently no way to specify the downloaded version. For this reason, when generating an image with Edge support users should only pass `--edge`.
 
+**note:** No major browsers are currently compatible with `arm64`, so browsers images will be missing those browsers on `linux/arm64` architecture. As time passes and these become available, we will introduce them to the `arm64` images as well: https://github.com/cypress-io/cypress-docker-images/issues/695.
```

Comment:
I probably don't understand this well enough but my understanding is at least Chrome has an arm64 build, no? https://support.google.com/chrome/thread/83557143

Or is this just a build that works on arm64 machines, via some kind of emulation layer (not a """real""" arm64 build).

Reply:
It's available for arm64... on darwin. `linux-arm64` is still not supported: https://bugs.chromium.org/p/chromium/issues/detail?id=677140 Since the Docker image is running on a Linux VM with a Linux kernel on Mac, we need a `linux-arm64` build to run in Docker for Mac.

Reply:
The logged date of that issue...

> Wed, Dec 28, 2016

Do not hold your breath, although hopefully the popularity of M1 will push this forward.